openjdk-crac-17.yaml

```diff
@@ -0,0 +1,300 @@
+package:
+  name: openjdk-crac-17
+  version: 17.0.25
+  epoch: 0
+  description:
+  copyright:
+    - license: GPL-2.0-only
+  dependencies:
+    runtime:
+      - openjdk-crac-17-jre
+
+var-transforms:
+  - from: ${{package.version}}
+    match: (\d+)\.\d+\.(\d+)
+    replace: $1+$2
+    to: mangled-package-version
+
+environment:
+  contents:
+    packages:
+      - alsa-lib-dev
+      - autoconf
+      - automake
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cups-dev
+      - file
+      - fontconfig-dev
+      - freetype-dev
+      - giflib-dev
+      - lcms2-dev
+      - libffi-dev
+      - libjpeg-turbo-dev
+      - libx11-dev
+      - libxext-dev
+      - libxi-dev
+      - libxrandr-dev
+      - libxrender-dev
+      - libxt-dev
+      - libxtst-dev
+      - openjdk-16
+      - openjdk-16-default-jvm
+      - zip
+      - crac-criu
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/openjdk/crac.git
+      tag: jdk-${{vars.mangled-package-version}}
+      expected-commit: 6765f902505fbdd02f25b599f942437cd805cad1
+
```

Comment:
I expect patched code to be build here. I.e. crac-17 branch or crac branch. As the tags seem to point at stock vanilla JDK.